geonode/maps/templates/maps/map_detail.html

```diff
@@ -207,7 +207,7 @@ <h4 class="list-group-item-heading">{% trans "Map Layers" %}</h4>
         <li class="list-group-item">
           <h4>{% trans "Copy this map" %}</h4>
           <p>{% trans "Duplicate this map and modify it for your own purposes" %}</p>
-          <a href="{% url "new_map" %}?copy={{ resource.id }}" class="btn btn-default btn-md btn-block">{% trans "Create a New Map" %}</a>
+          <a href="{% url "copy_map" resource.id %}" class="btn btn-default btn-md btn-block">{% trans "Create a New Map" %}</a>
```

Comment:
i think the link indeed broken from the beginning and it won't work with the geoserver backend as well.
i can't find the handler for the url link in the beginning.
that's why i created a new handler on the url.py